### ISSUE TITLE
Fix data type plot function

### ISSUE THREAD

**Issue by thomvolker** [thomvolker/densityratio#16]
``` r
library(densityratio)

set.seed(123)

x <- rnorm(200)
y <- rnorm(200, 1, sqrt(2))

fit <- ulsif(x, y, centers = x, sigma = 1, lambda = 0.5)
#> Warning in check.dataform(nu, de, df_centers, is.null(centers), NULL, scale):
#> Note that you provided centers while also applying scaling to the variables.
#> The centers are scaled accordingly.

plot(fit)
#> Error in rep("numerator", nrow(x$df_numerator)): invalid 'times' argument
```

<sup>Created on 2024-06-24 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by thomvolker** [comment]
Same error occurs in `plot_univariate()`.

**Comment by thomvolker** [comment]
Fixed in 56d855c4e877aee339e76eccea8d33324d3d0e6e.
